io-engine/src/bin/io-engine-client/v1/replica_cli.rs

```diff
@@ -139,6 +184,13 @@ pub async fn handler(
         ("share", Some(args)) => replica_share(ctx, args).await,
         ("unshare", Some(args)) => replica_unshare(ctx, args).await,
         ("stats", Some(args)) => replica_stat(ctx, args).await,
+        ("snapshot_create", Some(args)) => {
+            replica_snapshot_create(ctx, args).await
+        }
+        ("snapshot_list", Some(args)) => replica_snapshot_list(ctx, args).await,
```

Comment:
Snapshot listing and removal seem to have nothing related to replicas, as they just require UUID of the snapshot.
Snapshot removal seems to be a generic snapshot operation, so can we introduce a dedicated CLI command for it ? We could later generalise snapshot listing too.
```
> io-engine-client snapshot delete <UUID>
```

We could later also generalise our .proto files and move such generic operations there (currently snapshot listing/removal reside in replica-related .proto). So let's have CLI properly formed first.